### DISCUSSION TITLE
Trigger Deploy On Repo Release

### DISCUSSION THREAD

**Discussion by Mayank0255** [vercel/community#712] [Help] [answered]
I have a react repo with vercel deployment,

Currently if I push to master or merge to master, then it deploys itself, what I want is to do a staging deploy at that time, and a prod release will be triggered only if I make a repo release

**Comment by swarnava** [comment, marked as answer]
You can setup it via GitHub Action or custom CI Pipe and trigger the build using [Vercel CLI](https://vercel.com/docs/cli)
